drivers/spi/Kconfig.nrfx

```diff
@@ -162,13 +162,29 @@ endif # SPI_2
 
 if SPI_3
 
+choice
```

Comment:
Question: why this is not symmetric with the rest of SPI ports?
E.g. there is no config SPI_3_NRF_SPI

Reply:
Because there is no such peripheral in any of our SoCs. There are SPIs 0-2 in nRF51/52, and in nRF91 there are only SPIMs. I don't think it is worth adding such entry just for symmetry.